src/backend/columnar/Makefile

```diff
@@ -0,0 +1,17 @@
+citus_subdir = src/backend/columnar
+citus_top_builddir = ../../..
+safestringlib_srcdir = $(citus_abs_top_srcdir)/vendor/safestringlib
+SUBDIRS = . safeclib
+SUBDIRS +=
+ENSURE_SUBDIRS_EXIST := $(shell mkdir -p $(SUBDIRS))
+OBJS += \
+	$(patsubst $(citus_abs_srcdir)/%.c,%.o,$(foreach dir,$(SUBDIRS), $(sort $(wildcard $(citus_abs_srcdir)/$(dir)/*.c))))
+
+MODULE_big = citus_columnar
+
+PG_CPPFLAGS += -I$(libpq_srcdir) -I$(safestringlib_srcdir)/include
+
+include $(citus_top_builddir)/Makefile.global
+
+.PHONY: install-all
+install-all: install
```

Comment:
does this install downgrade scripts too?

Reply:
No, it won't install downgrade. I ran install-all for my branch vs master, the sql files under share/postgresql/extension are exactly the same, columnar behavior remains same as what we have on current master